api/api.go

```diff
@@ -226,6 +226,11 @@ func (api *ExtensionsApi) Notify(extensions []core.Extension) {
 			if err != nil {
 				log.Printf("failed to merge update data %v", err)
 			}
+			// manually overwite localization data
+			if castedData.Development.LocalizationStatus == "success" {
```

Comment:
I think it would be more accurate to compare  `api.Extensions[index].Localization.LastUpdated` with `castedData.Localization.LastUpdated` after we move the timestamp generation to the right place. This guarantees we update the localization data only when it's newer